swagger-notebook-service/jupyter-swagger-codegen/src/main/resources/jupyter-swagger-codegen/Dockerfile.mustache

```diff
@@ -3,7 +3,7 @@
 FROM jupyter/all-spark-notebook:dbb9c7b50531
 
 # Install Kernel Gateway
-RUN pip install git+https://github.com/jupyter-incubator/kernel_gateway.git
+RUN pip install --no-cache-dir git+https://github.com/jupyter-incubator/kernel_gateway.git
```

Comment:
Convert to straight pip install like the others - incubator is no longer required.

Reply:
Done as requested